docs/UserGuide/Maintenance-Tools/Maintenance-Command.md

```diff
@@ -179,6 +180,61 @@ IoTDB> show schema regions
 +--------+------------+------+-------------+------------+----------+----------+---------+----+
 Total line number = 2
 It costs 0.012s
+    
+IoTDB> show regions of storage group root.sg1
```

Comment:
The commands from `show regions of storage group root.sg1` to the end lacks the timeseries created statement.